docs/framework/concepts/yaml_format.md

```diff
@@ -0,0 +1,283 @@
+---
+short-description: Understanding the YAML format used to create apps
+...
+# The YAML format
+
+The Modular framework provides tools for creating knowledge apps by combining existing modules. App creators can combine these modules and describe the visual structure of their apps by writing YAML files that follow a specific format. This document is meant to explain the format of this YAML file. Here's the fist example:
```

Comment:
When writing Markdown I find it easier to review (and to read diffs) by breaking the line after each sentence. (It makes no difference in the rendered HTML.)

Typo: fist -> first